spec/support/helpers/oidc_client_helper.rb

```diff
@@ -30,6 +30,18 @@ def stub_token_response
     allow_any_instance_of(OidcClient).to receive(:tokens!).and_return(token_response)
     # rubocop:enable RSpec/AnyInstance
   end
+
+  def stub_remote_attribute_request(name:, value: nil, status: nil)
+    status ||= value.nil? ? 404 : 200
+    stub_request(:get, "http://openid-provider/v1/attributes/#{name}")
+      .to_return(status: status, body: { claim_value: value }.compact.to_json)
+  end
+
+  def stub_remote_attribute_requests(names_and_values = {})
+    names_and_values.each do |name, value|
+      stub_remote_attribute_request(name: name, value: value)
```

Comment:
So we're happy that these can only ever 200?
I guess this is the most high-use stub for a happy path?

Thinking out loud:
If we wanted to test 422 or 404 across multiple (... would we?) we can write a specific stub for those...

Reply:
Yeah, we could always make a new stub for a situation like that, or extend this one